.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.2
```

Comment:
❤️ a Gemfile would be nice to, to include rubocop and rspec in the future

Reply:
nice!!